vtex/admin-ui
# NAD-866 - Alert component review

#### What is the purpose of this pull request?

To update the current Alert component following the component review guidelines.

[Design specs are available on this Figma file.](https://www.figma.com/file/sepBhrHViUHyrTnktkuHaW/Admin-UI-%E2%80%93-Components?node-id=656%3A286721)

#### How should this be manually tested?


[Visit this storybook story ](https://alert-review.surge.sh/?path=/story/admin-ui-review-alert--tones)and its [documentation section](https://admin-ui-docs-git-feat-alert-review-admin-team.vercel.app/next/guidelines/components/alert). Make sure it matches the Figma specs.

### Breaking changes

- The `icon`, `fluid`, `visible`, and `sticky` properties were removed. 
- The `fluid` behavior was incorporated as default for it.
- Each variant has its own default icon now.
- The `tone` prop was renamed to `variant`.

You can use our codemod cli to update the alert's properties for you. [See its README for more information on how to execute the codemod](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-codemod#admin-ui-codemod).

#### Before
```tsx
import { Alert } from '@vtex/admin-ui'

function Example() {
  return (
    <Alert
      icon={<IconHeart />}
      fluid
      sticky
      visible
    >Hello world</Alert>
  )
}
```

#### After
```tsx
import { Alert } from '@vtex/admin-ui'

function Example() {
  return <Alert visible>Hello world</Alert>
}
```

#### Types of changes
